### PR TITLE
task/handle-retrofit-imu-reset

### PR DESCRIPTION
**Pull request by ferrom-jaia**
# Summary
Adding configuration for imu installation type
# Details
The approach for the imu installation type is similar to the imu type configuration 
# Testing
The GPIO pin for both the retrofit and the embedded IMU has been tested
# Notes
Please merge this pull request in first: https://github.com/jaiarobotics/jaiabot-debian/pull/53

### CODE REVIEW

File: src/python/adafruit_BNO085/jaia_firm_bno085_reset_gpio_pin.py [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
+import argparse
 
-# BNO085 IMU reset pin
-bno085_pin = 10
+parser = argparse.ArgumentParser(description='Turn on/off bno085 imu', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument('--imu_install_type', choices=['embedded', 'retrofit', 'none'], help='If set, configure services for imu install type')
+args=parser.parse_args()
+
+# Set BNO085 IMU reset pin
+if args.imu_install_type == 'embedded':
```
Comment (michael-jaia) [inline src/python/adafruit_BNO085/jaia_firm_bno085_reset_gpio_pin.py]: Not sure if it's important here, but could could we remove the elif by only checking for retrofit with the if and then everything else is 10

**Review by michael-jaia — COMMENTED** [review]
Just left one comment...not sure if it needs to be changed just something I noticed. I also did not know variables defined within python if statements are not closed scope to within that block

**Comment by ferrom-jaia** [comment]
> Just left one comment...not sure if it needs to be changed just something I noticed. I also did not know variables defined within python if statements are not closed scope to within that block

Ya, I think we can move the variable outside to make it more readable

File: src/python/adafruit_BNO085/jaia_firm_bno085_reset_gpio_pin.py (hunk 1 above)
Comment (ferrom-jaia) [reply src/python/adafruit_BNO085/jaia_firm_bno085_reset_gpio_pin.py]: Yes, I will update

**Review by michael-jaia — APPROVED** [review]
Comments resolved, looks good, approving PR into 1.y

**Pull request merged**